### ISSUE TITLE
ng add @clr/angular@next fails with Angular9

### ISSUE THREAD

**Issue by mcosta74** [vmware/clarity#4277]
## Describe the bug

```
ng add @clr/angular@next
```
fails on a fresh new Angual9 project

## How to reproduce

1.  Create a new Angular9 project with `ng add my-project`
2.  Add clarity 3 next with `ng add @clr/angular@next`

I get this error
```
An unhandled exception occurred: Collection "@clr/angular" cannot be resolved.
See "/private/var/folders/xv/8mqxb9qx61bbj7bvxgdv9vmm0000gq/T/ng-II6mmk/angular-errors.log" for further details.
```
[angular-errors.log](https://github.com/vmware/clarity/files/4204350/angular-errors.log)

## Expected behavior

Everything goes well

## Versions

**App**

* Angular: 9.0.1
* Angular CLI: 9.0.2
* Node: 10.13.0
* Clarity: 3.0@next0

**Device:**

* Type: MacBook
* OS: macOS

**Comment by gnomeontherun** [comment]
This is a known issue due to a publishing error, it will be resolved in the next release. Essentially the schematics weren't published with the package.

**Comment by gnomeontherun** [comment]
Fixed by #4262. Will be fixed in next release.
